airbyte-integrations/connectors/source-spacex-api/source_spacex_api/schemas/launches.json

```diff
@@ -0,0 +1,230 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "fairings": {
+      "type": "null"
+    },
+    "links": {
+      "type": "object",
+      "properties": {
+        "patch": {
+          "type": "object",
+          "properties": {
+            "small": {
+              "type": "string"
+            },
+            "large": {
+              "type": "string"
+            }
+          },
+          "required": ["small", "large"]
+        },
+        "reddit": {
+          "type": "object",
+          "properties": {
+            "campaign": {
+              "type": "null"
+            },
+            "launch": {
+              "type": "string"
+            },
+            "media": {
+              "type": "null"
+            },
+            "recovery": {
+              "type": "null"
+            }
+          },
+          "required": ["campaign", "launch", "media", "recovery"]
+        },
+        "flickr": {
+          "type": "object",
+          "properties": {
+            "small": {
+              "type": "array",
+              "items": {}
+            },
+            "original": {
+              "type": "array",
+              "items": {}
+            }
+          },
+          "required": ["small", "original"]
+        },
+        "presskit": {
+          "type": "null"
+        },
+        "webcast": {
+          "type": "string"
+        },
+        "youtube_id": {
+          "type": "string"
+        },
+        "article": {
+          "type": "null"
+        },
+        "wikipedia": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "patch",
+        "reddit",
+        "flickr",
+        "presskit",
+        "webcast",
+        "youtube_id",
+        "article",
+        "wikipedia"
+      ]
+    },
+    "static_fire_date_utc": {
+      "type": "null"
+    },
+    "static_fire_date_unix": {
+      "type": "null"
+    },
+    "net": {
+      "type": "boolean"
+    },
+    "window": {
+      "type": "null"
+    },
+    "rocket": {
+      "type": "string"
+    },
+    "success": {
+      "type": "boolean"
+    },
+    "failures": {
+      "type": "array",
+      "items": {}
+    },
+    "details": {
+      "type": "null"
+    },
+    "crew": {
+      "type": "array",
+      "items": [
+        {
+          "type": "string"
+        },
+        {
+          "type": "string"
+        },
+        {
+          "type": "string"
+        },
+        {
+          "type": "string"
+        }
+      ]
+    },
+    "ships": {
+      "type": "array",
+      "items": {}
+    },
+    "capsules": {
+      "type": "array",
+      "items": [
```

Comment:
"items" must be an object instead of an array :)

Reply:
Thanks for the help! Acceptance tests passed, Cheers!